static.config.js

```diff
@@ -644,6 +653,41 @@ const buildPageAtUrl = async (pageAtUrlInfo, client) => {
   }
 };
 
+const getPagesOfGuidesData = async (client) => {
+
+  const { allGuidePages } = await client.request(getAllGuidePagesSectionsQuery)
+
+  const pagesOfGuidesData = {}
+
+  allGuidePages.edges.map( guidePage => {
+    if (guidePage.node.sections.length > 0 && guidePage.node.topics.edges.length > 0) {
+      const url = "/" + [
+        guidePage.node.topics.edges[0].node.topic.topiccollections.edges[0].node.topiccollection.theme.slug,
+        guidePage.node.topics.edges[0].node.topic.topiccollections.edges[0].node.topiccollection.slug,
+        guidePage.node.topics.edges[0].node.topic.slug,
+        guidePage.node.slug,
+      ].join("/")
+      guidePage.node.sections.map( section => {
+        for (const page in section.pages[0]) {
+          if (section.pages[0][page]) {
+            if (!pagesOfGuidesData[page]) pagesOfGuidesData[page] = {}
+            if (!pagesOfGuidesData[page][section.pages[0][page].id]) pagesOfGuidesData[page][section.pages[0][page].id] = []
+            pagesOfGuidesData[page][section.pages[0][page].id].push({
+              pageName: section.heading,
+              pageType: section.pages[0][page].pageType,
+              ofPageType: guidePage.node.pageType,
+              guidePageTitle: guidePage.node.title,
+              guidePageUrl: url
+            })
+          }
+        }
+      })
+    }
+  })
+
+  return pagesOfGuidesData
+}
+
```

Comment:
This is where we query and parse all the guide pages and create  an objectKey for all the IDs of the service and Info pages to have a reference to the guides(s) they're a part of.  

At this point, it'll work for anyOther type of page, but what It'll look like now is just { service:{}, information: {} }

Then within those pages are key's names as Id, which is what will be checked against when building the service and info pages. 

Let me know if there are any other Qs.